lib/picos_rc/picos_rc.ml

```diff
@@ -0,0 +1,47 @@
+type 'a t = { count : int Atomic.t; value : 'a; dispose : 'a -> unit }
+
+let one = 2
```

Comment:
oh, I see, it's a bitfield. There should be a comment explaining the use of the LSB! And I like the notation `0b10` here to underline that!

Reply:
Yes, I should add a comment there.  The lowest bit, i.e. `alive_bit`, indicates whether the resource is alive or has been marked for disposal.  The idea is that dispose is an idempotent decrement, i.e. it decrements exactly once (i.e. clears the `alive_bit`), so multiple entities may concurrently decide to dispose.  This design (API) is somewhat WIP still.  I plan to go through it at least once more and think through how it really should work to support the use cases.  (The changes to the `Async_unix` module are not correct at the time of writing this - I just edited the code so that it compiles.)

Reply:
I added a comment there.